README.md

```diff
@@ -46,12 +48,12 @@ We're creating a lasting team built on teamwork and collaboration. [It takes a v
 
 The Engineering team is committed to providing a welcoming and safe environment for all people -- people of all races, gender identities, gender expressions, sexual orientations, physical abilities, physical appearances, socioeconomic backgrounds, nationalities, ages, religions, and beliefs.
 
-We have two-way, transparent, and ethical conversations. [Always honest](https://). Any behavior or language that is unwelcoming is strongly discouraged. Such exclusionary behavior can take many forms, including [microaggressions](http://www.vox.com/2015/2/16/8031073/what-are-microaggressions)—subtle put-downs, whether deliberate or unconscious. Regardless of intent, microaggressions can have a significant negative impact on victims and have no place on our team.
+We have two-way, transparent, and ethical conversations. [Always honest](https://). Any behavior or language that is unwelcoming is strongly discouraged. Such exclusionary behavior can take many forms, including [microaggressions](http://www.vox.com/2015/2/16/8031073/what-are-microaggressions)—-subtle put—-downs, whether deliberate or unconscious. Regardless of intent, microaggressions can have a significant negative impact on colleagues and have no place on our team.
```

Comment:
💯 for removing "victims". That extra m-dash in the middle of "put-downs" we don't want.